packages/generator-clay-component/app/templates/_package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "@clayui/<%= repoName %>",
+	"version": "3.0.0",
+	"description": "<%= componentName %> component",
+	"license": "BSD-3-Clause",
+	"repository": "https://github.com/liferay/clay/tree/master/packages/<%= repoName %>",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "lib/<%= componentName %>.js",
+	"esnext:main": "src/<%= componentName %>.js",
+	"jsnext:main": "src/<%= componentName %>.js",
+	"files": ["lib", "src"],
+	"scripts": {
+		"build": "babel src --root-mode upward --out-dir lib --extensions .ts,.tsx",
+		"build:types": "tsc --project ./tsconfig.declarations.json",
+		"prepublish": "npm run build && npm run build:types",
```

Comment:
This should probably be "prepublishOnly", right?

https://docs.npmjs.com/misc/scripts